updatecli/updatecli.d/nodejs.yml

```
@@ -54,7 +54,7 @@ targets:
     sourceid: lastReleaseVersion
     kind: yaml
     spec:
-      file: goss/goss.yaml
+      file: goss/goss-linux.yaml
       key: $.command.nodejs.stdout[0]
```

```suggestion
      file: goss/goss-linux.yaml
      key: $.command.nodejs_version.stdout[0]
```

Ref. https://github.com/jenkins-infra/packer-images/pull/852/files#diff-d450d3124af9f00269d6422ecdbbf49ec01664e645d73ddcfa4a89984e5c6ab7R35